crates/forge/bin/main.rs

```diff
@@ -19,7 +19,8 @@ static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
 fn main() -> Result<()> {
     handler::install();
-    utils::load_dotenv();
+    let config = utils::load_config();
+    utils::load_dotenv(&config.env_path);
```

Comment:
this complicates things a lot, because is an additional config lookup, which isn't super cheap.

I also don't want to automatically load env vars when loading the config.
so not quite sure yet where to apply this